src/main/java/uk/gov/hmcts/reform/logging/exception/AbstractLoggingException.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.logging.exception;
+
+public abstract class AbstractLoggingException extends RuntimeException {
+
+    public AlertLevel alertLevel = AlertLevel.P1;
```

Comment:
does this have to be public and mutable?

Reply:
it is used outside of the package/inheritance scope - when formatting the message. mutability is required to set different alert level (P2, P3 or P4). alternatively, can create four abstracts with static members and get rid of an enum. would not want to go further than that as it already sounds like over-designing

Reply:
can we not use a getter for it (and no setter)?